keyboards/null/st110r2/info.json

```
@@ -0,0 +1,155 @@
+{
+    "manufacturer": "qmk",
```

this should probably be something other than "qmk". looking at the url provided it's probably 
```suggestion
    "manufacturer": "[null]",
```